vulns/CVE-2023-5044.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "CVE-2023-5044",
+  "modified": "2023-10-25T15:48:28Z",
+  "published": "2023-10-25T15:48:28Z",
+  "summary": "Code injection via nginx.ingress.kubernetes.io/permanent-redirect annotation",
+  "details": "Code injection via nginx.ingress.kubernetes.io/permanent-redirect annotation.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/ingress-nginx"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:L/A:L"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "0"
```

Comment:
`v1.9.0` is not fixed version:
```
Versions allowing mitigation

v1.9.0
Mitigation

Ingress Administrators should set the --enable-annotation-validation flag to enforce restrictions on the contents of ingress-nginx annotation fields.
```

See https://github.com/kubernetes/kubernetes/issues/126817